optimize/backend/src/main/java/io/camunda/optimize/service/importing/engine/mediator/TenantImportMediator.java

```diff
@@ -1,64 +0,0 @@
-/*
- * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
- * one or more contributor license agreements. See the NOTICE file distributed
- * with this work for additional information regarding copyright ownership.
- * Licensed under the Camunda License 1.0. You may not use this file
- * except in compliance with the Camunda License 1.0.
- */
-package io.camunda.optimize.service.importing.engine.mediator;
-
-import io.camunda.optimize.dto.engine.TenantEngineDto;
-import io.camunda.optimize.service.importing.BackoffImportMediator;
-import io.camunda.optimize.service.importing.engine.fetcher.instance.TenantFetcher;
-import io.camunda.optimize.service.importing.engine.handler.TenantImportIndexHandler;
-import io.camunda.optimize.service.importing.engine.service.TenantImportService;
-import io.camunda.optimize.service.util.BackoffCalculator;
-import io.camunda.optimize.service.util.configuration.ConfigurationService;
-import java.util.List;
-import org.springframework.beans.factory.config.ConfigurableBeanFactory;
-import org.springframework.context.annotation.Scope;
-import org.springframework.stereotype.Component;
-
-@Component
-@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
-public class TenantImportMediator
-    extends BackoffImportMediator<TenantImportIndexHandler, TenantEngineDto> {
```

Comment:
Yeah, though I think it would be useful to first clarify question 2 (using the word engine in eg`ENGINE_ALIAS_OPTIMIZE`) as well as whether we're keeping event based stuff. If we were to remove event based stuff this mediator would only be relevant for external variable ingestion at which point we would probably refactor this differently than if we keep event based stuff. 
Theres also some tenant exclusion logic which I need to clarify on whether this is also a (future) C8 feature or not (plus if we're keeping it, need to review the `engine` terminology again in `TenantSpecificEngineDto`)

Reply:
Actually, looking at it again now seems like we're already not doing tenant specific import filtering for events or external variables, so its not even a C7 feature for non "Engine" imports. In that case I think its safe to assume we can throw it out for C8 too will remove 👍